CHANGELOG.md

```diff
@@ -76,7 +80,7 @@ All notable changes to this project will be documented in this file.
 ## [1.3.3]
 * Feature:
 	* Seek media with a given offset
-	* Seek position is bounded between 0 and current media end time 
+	* Seek position is bounded between 0 and current media end time
```

Comment:
diffs should not be present here and below

Reply:
Sorry, Atom automatically removes whitespaces at the end of the line. Reverted